src/test/java/uk/gov/hmcts/reform/pip/data/management/service/filegeneration/nonstrategic/UtIacJudicialReviewLeedsDailyHearingListFileConverterTest.java

```diff
@@ -0,0 +1,274 @@
+            .containsExactly(
+                HEARING_VENUE,
+                "Judge A",
+                "10:30am",
+                "1234",
+                "Case A",
+                "Hearing type A",
+                HEARING_ADDITIONAL_INFORMATION,
+                HEARING_VENUE,
+                "Judge B",
+                "11am",
+                "5678",
+                "Case B",
+                "Hearing type B",
+                HEARING_ADDITIONAL_INFORMATION
+            );
+
+        softly.assertAll();
+    }
+
+    @Test
+    void testUtIacJudicialReviewDailyHearingListFileConversionInWelsh() throws IOException {
+        Map<String, Object> languageResource;
+        try (InputStream languageFile = Thread.currentThread()
+            .getContextClassLoader()
+            .getResourceAsStream("templates/languages/cy/non-strategic/utIacJrLeedsDailyHearingList.json")) {
+            languageResource = new ObjectMapper().readValue(
+                Objects.requireNonNull(languageFile).readAllBytes(), new TypeReference<>() {
+                });
+        }
+
+        Map<String, String> metadata = Map.of(CONTENT_DATE_METADATA, CONTENT_DATE,
+                                              PROVENANCE_METADATA, PROVENANCE,
+                                              LANGUAGE_METADATA, WELSH,
+                                              LIST_TYPE_METADATA, UT_IAC_JR_LEEDS_DAILY_HEARING_LIST.name(),
+                                              LAST_RECEIVED_DATE_METADATA, LAST_RECEIVED_DATE
+        );
+
+        String result = converter.convert(utIacJudicialReviewInputJson, metadata, languageResource);
+        Document document = Jsoup.parse(result);
+
+        SoftAssertions softly = new SoftAssertions();
+
+        softly.assertThat(document.title())
+            .as(TITLE_MESSAGE)
+            .isEqualTo("Uwch Dribiwnlys (Mewnfudo a Lloches) - "
+                   + "Rhestr o Wrandawiadau Dyddiol Siambr Adolygiadau Barnwrol Leeds");
+
+        softly.assertThat(document.getElementById(HEADER_ELEMENT).text())
+            .as(HEADER_MESSAGE)
+            .isEqualTo("Uwch Dribiwnlys (Mewnfudo a Lloches) - "
+                   + "Rhestr o Wrandawiadau Dyddiol Siambr Adolygiadau Barnwrol Leeds");
+
+        softly.assertThat(document.getElementById(LIST_DATE_ELEMENT).text())
+            .as(LIST_DATE_MESSAGE)
+            .isEqualTo(LIST_DATE_WELSH);
+
+        softly.assertThat(document.getElementById(LAST_UPDATED_DATE_ELEMENT).text())
+            .as(LAST_UPDATED_DATE_MESSAGE)
+            .isEqualTo("Diweddarwyd ddiwethaf 20 January 2025 am 9:30am");
+
+        softly.assertThat(document.getElementsByClass(SUMMARY_TEXT_CLASS).get(0).text())
+            .as(IMPORTANT_INFORMATION_MESSAGE)
+            .isEqualTo("Gwybodaeth bwysig");
+
+        softly.assertThat(document.getElementById(LIST_UPDATE_MESSAGE_ELEMENT).text())
+            .as(BODY_MESSAGE)
+            .contains("Gall y rhestr ganlynol newid tan 4:30pm.");
```

Comment:
It will work but can you add full open justice wording to make it consistent with other lists.